### DISCUSSION TITLE
Multiple heating zones and current power consumption integration maybe possible?

### DISCUSSION THREAD

**Discussion by Flori1007** [signalkraft/mypyllant-component#120] [Ideas]
Unfortunately, there is still no way to display multiple heating circuits. I have a PV system and have therefore created a second simulating heating circuit. This only has the purpose of increasing the water temperature. Unfortunately, I cannot display this. 

It is also not yet possible to display the current energy consumption. Is this possibly planned for your next updates? In the MyVaillant app, this can already be displayed and shows the current power consumption almost to the second. 

Translated with DeepL.com (free version)

**Comment by lander2304** [comment]
Yea I realy would like to now to current enrgy consumption of my system. I have been looking for a way but haven't foond one. Does any one now a way to import this into home assistant or node-red? How they import this in a futur update in mypyllant.
